.github/workflows/build_package.yml

```
@@ -0,0 +1,17 @@
+name: Build Maven Package
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
```

Is this for backward compability reasons? I thought we agreed on JDK 11
```suggestion
          java-version: 1.11
```